### DISCUSSION TITLE
Unit Test Mocking not working

### DISCUSSION THREAD

**Discussion by mhuggins7278** [prisma/prisma#12431] [Q&A] [answered]
Following the instructions here https://www.prisma.io/docs/guides/testing/unit-testing
I am unable to get jest to correctly mock the prisma client. 
The tests run but require the db to be running any insights. 

Here is a link to my repo if that's helpful. https://github.com/mhuggins7278/learn-build-teach-discord-bot/tree/fixTests

**Comment by mhuggins7278** [comment, marked as answer]
I've resolved my issue. I accidentally had 2 client.ts files and was mocking the wrong one.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
